src/server/config.js

```diff
@@ -56,6 +56,9 @@ export default function (baseConfig, configDir) {
     module: {
       ...config.module,
       // We need to use our and custom loaders.
+      preLoaders: [
```

Comment:
I think we need on each loader, because if we default to empty object it won't use default config.